src/SymfonyCastsVerifyEmailBundle.php

```diff
@@ -9,22 +9,37 @@
 
 namespace SymfonyCasts\Bundle\VerifyEmail;
 
-use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
-use Symfony\Component\HttpKernel\Bundle\Bundle;
-use SymfonyCasts\Bundle\VerifyEmail\DependencyInjection\SymfonyCastsVerifyEmailExtension;
+use Symfony\Component\Config\Definition\Configurator\DefinitionConfigurator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Component\HttpKernel\Bundle\AbstractBundle;
 
 /**
  * @author Jesse Rushlow <jr@rushlow.dev>
  * @author Ryan Weaver   <ryan@symfonycasts.com>
  */
-class SymfonyCastsVerifyEmailBundle extends Bundle
+class SymfonyCastsVerifyEmailBundle extends AbstractBundle
 {
-    public function getContainerExtension(): ?ExtensionInterface
+    protected string $extensionAlias = 'symfonycasts_verify_email';
+
+    public function configure(DefinitionConfigurator $definition): void
+    {
+        $definition->rootNode()
+            ->children()
+                ->integerNode('lifetime')
+                    ->defaultValue(3600)
+            ->info('The length of time in seconds that a signed URI is valid for after it is created.')
```

Comment:
Is the line indent missing here?

Reply:
note to self: check php-cs-fixer rules to see if its possible to flag this in the future...